src/goto-instrument/dump_c.cpp

```diff
@@ -1062,6 +1178,8 @@ void dump_ct::convert_function_declaration(
   if(symbol.name!=goto_functionst::entry_point() &&
      symbol.name!=ID_main)
   {
+    collect_typedefs(symbol.type, true);
```

Comment:
This comment still seems relevant: How come we don't collect typedef's for the main function?